src/utils/extract-from-document.js

```diff
@@ -18,6 +18,12 @@ async function findHTMLFiles(directory) {
   return htmlFiles;
 }
 
+async function unzipFileToDirectory(zipFilePath, destinationFolder) {
+  const zip = new AdmZip(zipFilePath);
+
+  zip.extractAllTo(destinationFolder);
```

Comment:
I think extractAllTo is a synchronous OP. Did you mean to use zip.extractAllToAsync instead? (It's not documented in the admzip docs but it's a thing)

Reply:
Hi Armaan, thanks for your comment! I think I should remove `async` from the `unzipFileToDirectory` function, it's ok for it to be synchronous here as it needs to unzip the file before the content can be extracted.